template/cordova/lib/AppxManifest.js

```diff
@@ -374,6 +374,11 @@ AppxManifest.prototype.getCapabilities = function () {
 };
 
 function refineColor(color) {
+    if (color.indexOf('0x') === -1 && color.indexOf('#') === -1) {
```

Comment:
Is there a list of valid CSS colours? For example, `meow` and `12345` and are not colours, but they will get accepted here.